css/style.css

```diff
@@ -0,0 +1,589 @@
+  width: 80px;
+  height: 35px;
+  opacity: 0.7;
+}
+
+h3 {
+  font-size: 24px;
+  line-height: 24px;
+}
+
+.image-thumbnail {
+  width: 63px;
+}
+
+#versus-logo {
+  width: 30px;
+}
+
+.email-subscription,
+.email-subscription-top {
+  background-color: #f4f5f6;
+}
+
+#versus-container::before {
+  content: "";
+  display: block;
+  background-color: #d9d9d9;
+  height: 1px;
+  position: absolute;
+  left: 0;
+  right: 0;
+}
+
+#versus-container::after {
+  content: "";
+  display: block;
+  background-color: #d9d9d9;
+  height: 1px;
+  position: absolute;
+  left: 0;
+  right: 0;
+}
+
+.horizontal-line {
+  z-index: -1;
+  top: 15px;
+  height: 3px;
+}
+
+.svg-image {
+  z-index: 50;
+  top: 60px;
+}
+
+.right-news-cont {
+  display: none;
+}
+
+.section-logo {
+  width: 29px;
+}
+
+#newsletter-img-top {
+  width: 177px;
+}
+
+#footer-content {
+  flex-direction: column;
+}
+
+#footer-logo {
+  width: 251px;
+}
+
+.fa {
+  color: #656665;
+}
+
+#nl-img-cont {
+  height: 180px;
+}
+
+#newsletter-large-pic {
+  display: none;
+}
+
+div.smalldiv {
+  border: none !important;
+}
+
+@media screen and (max-width: 768px) and (min-width: 426px) {
+  .navbar-toggler-icon {
+    display: inline;
+  }
+
+  .feature-section,
+  .story-section,
+  .debate-section {
+    height: auto;
+  }
+
+  .feature-section,
+  .more-stories1 {
+    position: -webkit-sticky;
+    position: sticky;
+    top: 4rem;
+  }
+
+  .feature-cont {
+    align-items: flex-start;
+  }
+
+  .image-thumbnail1 {
+    margin-left: auto;
+  }
+
+  .last-section {
+    flex-direction: row;
+  }
+
+  .email-subscription-top {
+    width: 40%;
+    height: auto;
+    display: inline;
+  }
+
+  .email-subscription {
+    width: 100%;
+    height: auto;
+  }
+
+  .right-news-cont {
+    display: none;
+  }
+
+  .more-stories1 {
+    position: -webkit-sticky;
+    position: sticky;
+    bottom: 1rem;
+    align-self: flex-end;
+  }
+
+  #newsletter-large-pic {
+    display: inline;
+    background: url("https://d.newsweek.com/en/full/1541302/starting5-homepage.png?w=397&h=265&q=90&f=742bce9a6f72dd8f19ba2e7fd5db06e5");
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: cover;
+    height: 254px;
+  }
+
+  #sticky-footer {
+    margin: 0;
+    padding: 0;
+  }
+
+  .footer-copyright,
+  .social {
+    align-self: flex-end;
+    margin-right: auto;
+  }
+
+  #footer-content {
+    flex-direction: row;
+  }
+
+  .footer-copyright {
+    align-self: flex-end;
+  }
+}
+
+@media screen and (max-width: 991px) {
+  #middle-navbar,
+  nav.nav-navigation,
+  nav.navia {
+    display: none;
+  }
+}
+
+@media screen and (min-width: 992px) {
+  main {
+    margin-top: 20px;
+  }
+
+  nav.navia {
+    position: relative;
+    z-index: -1;
+  }
+
+  header.header-container {
+    position: -webkit-sticky; /* Safari */
+    position: sticky;
+    top: 0;
+  }
+
+  .top-header {
+    padding: 0;
+  }
+
+  #middle-navbar {
+    display: block;
+  }
```

Comment:
- [ ] There is a bootstrap class for this behavior, please use bootstrap whenever it is possible.